scripts/processAndDownloadContracts.ts

```diff
@@ -36,7 +36,9 @@ export const contractMetadata = new Map<string, ContractDefinition>(
 )
 `
 	const jsonData = JSON.stringify(contractData.map(( x) => [addressString(x.address), x.data.name, x.data.protocol, x.data.logoUri]), null, '\t')
-	fs.writeFileSync(`${OUTPUT_SRC_DIR}/contractMetadata.json`, jsonData, 'utf-8')
+	const tsJsonData = `export default ${jsonData};`
```

Comment:
Don't use default exports.  They exist for legacy reasons, but are generally considered bad.  `export const data = { ... } as const` or
```ts
export type ContractMetadata = { ... }
export const contractMetadata: ContractMetadata = { ... }
```

Reply:
Same comment elsewher in this PR (e.g., NFTs)

Reply:
```ts
export type Address = `0x${string}`
export type Name = string
export type Protocol = 'token' | 'nft'
export type LogoRelativePath = './${string}
export type ContractMetadata = [Address, Name, Protocol, LogoRelativePath][]
```

Reply:
I'm trying to figure out how to get a chance to get the money owed to me from this page is Jason package delivered to my bank account details for refund of money for the payment of my life and I have not been able to get my bank account details into the app